### PR TITLE
Add local env for passwords

### PR DESCRIPTION
**Pull request by michalina-graczyk**
I want to merge this change because this PR hides all test login passwords.

### Screenshots
image

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by michalina-graczyk** [comment]
The build is failing, because CI doesn't have .env set up yet. Waiting for @krzysztofwolski  to fix that.

File: cypress/support/login/index.js [hunk 1]
```diff
@@ -26,10 +26,10 @@ Cypress.Commands.add("loginUser", (email, password) => {
     .get(emailAddressInput)
     .type(email)
     .get(emailPasswordInput)
-    .type(password)
+    .type(password, { log: false })
     .get(signInButton)
     .click()
-    .get(allertPopupMessage, { timeout: 3000 })
+    .get(allertPopupMessage, { timeout: 30000 })
```
Comment (krzysztofwolski) [inline cypress/support/login/index.js:32]: Is `allert` a typo?

Comment (krzysztofwolski) [reply cypress/support/login/index.js:32]: Can we set up global timeout? Imo if we wait for server response more than 20sec then there is something not right.

Comment (michalina-graczyk) [reply cypress/support/login/index.js:32]: The alert is for "You are now logged in" message, you are right there is a typo.

Comment (michalina-graczyk) [reply cypress/support/login/index.js]: Timeouts removed, typo fixed.

**Pull request merged**